scripts/js/020-components/08-print.js

```diff
@@ -0,0 +1,23 @@
+// track print events in Google Analytics
```

Comment:
Seems like a waste of time to make analytics for print events. I'm sure users would benefit from other features.

Reply:
This was part of the acceptance criteria. We would like to see how much print is being used. Wasn't much work.